app/assets/javascripts/widgets/datepicker.js.coffee

```diff
@@ -13,7 +13,7 @@ widgets.define 'datepicker', (el) ->
     $el.datetimepicker
       format: DATE_FORMAT
       displayFormat: DATE_DISPLAY_FORMAT
-      locale: 'fr'
+      locale: $('html').attr('lang') or 'en'
```

Comment:
You can probably store it once locally so that we don't query the DOM 3 time when one is enough.